cmd/keytransparency-server/main.go

```diff
@@ -28,10 +28,14 @@ import (
 	"github.com/google/keytransparency/core/crypto/vrf/p256"
 	"github.com/google/keytransparency/core/keyserver"
 	"github.com/google/keytransparency/core/mapserver"
+	cmutation "github.com/google/keytransparency/core/mutation"
 	"github.com/google/keytransparency/core/mutator/entry"
 	ctxn "github.com/google/keytransparency/core/transaction"
+
 	gauth "github.com/google/keytransparency/impl/google/authentication"
+	"github.com/google/keytransparency/impl/mutation"
 	pb "github.com/google/keytransparency/impl/proto/keytransparency_v1_service"
```

Comment:
for consistency, please rename `pb` to `ktpb` (key transparency protobuf) no that it is no longer the only protobuf being included.